sickbeard/tv.py

```diff
@@ -186,6 +186,24 @@ def __init__(self, indexer, indexerid, lang='', quality=None, flatten_folders=No
         self._load_from_db()
 
     @property
+    def season_episodes_list(self, season):
+        """Returns the episodes numbers from season
+        
```

Comment:
First want to say nice job fernandog.  Second yes I think you should return list of TVEpisode() to do it proper.  As other property methods,  also return objects.